server/api/users.js

```diff
@@ -1,23 +1,57 @@
-const express = require('express');
+const express = require('express')
 
-const router = express.Router();
+const router = express.Router()
 
 const {
-  models: { User, Review },
-} = require('../db');
-
-router.get('/', async (req, res) => {
-  const users = await User.findAll({ include: Review });
-  res.send(users);
-});
-// get single user by specifying their email address
-router.get('/:id', async (req, res) => {
-  const user = await User.findOne({
-    where: {
-      id: req.params.id,
-    },
-  });
-  res.send(user);
-});
-
-module.exports = router;
+    models: { User, Review },
+} = require('../db')
+
+router.get('/', async (req, res, next) => {
+    try {
+        const users = await User.findAll({ include: Review })
+        res.send(users)
+    } catch (error) {
+        next(error)
+    }
+})
+
+router.get('/:id', async (req, res, next) => {
```

Comment:
You should include Review in here as well

Reply:
Got it - I'll add that in.

Reply:
We can talk about while we are all together but we might also want to include Orders? Maybe something you can @kfless12 can discuss